networks:
  auth_internal:
    external: true
  traefik:
    name: traefik
    external: true


services:

  portainer:
    image: portainer/portainer-ee:2.20.3
    restart: unless-stopped
    command: -H unix:///var/run/docker.sock --data=/mnt/ssd-1tb/docker/portainer
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - unused_data:/data
      - /mnt/ssd-1tb/docker/portainer:/mnt/ssd-1tb/docker/portainer
    networks:
      - auth_internal
      - traefik
    environment:
      TZ: "Europe/Berlin"
    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.portainer.rule=Host(`portainer.patz.app`)"
      - "traefik.http.routers.portainer.entrypoints=websecure"
      - "traefik.http.routers.portainer.service=portainer"
      - "traefik.http.routers.portainer.tls.certresolver=patz.app"
      - "traefik.http.routers.portainer.tls.domains[0].main=patz.app"
      - "traefik.http.routers.portainer.tls.domains[0].sans=portainer.patz.app"
      - "traefik.http.services.portainer.loadbalancer.server.port=9000"

